Application/EdFi.Ods.SandboxAdmin/Services/IdentityContextFactory.cs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.SandboxAdmin.Contexts;
+
+namespace EdFi.Ods.Sandbox.Admin.Services
+{
+    // TODO ODS-4538
+    public class IdentityContextFactory : IIdentityContextFactory
```

Comment:
@jamessmckay  can you leave this under contexts folder, that is where rest of the context related classes are.